### ISSUE TITLE
How should chapter inform waiting users after banning them

### ISSUE THREAD

**Issue by Sboonny** [freeCodeCamp/chapter#1767]
At #1729, banned users lose the RSVP they had. Here is the function: [updateEventWaitlist](https://github.com/freeCodeCamp/chapter/blob/750bfa3a134b73aa3edaf6942a76bc0a1b7ea677/server/src/util/updateEventWaitlist.ts#L14-L53?plain=1)

How should we inform them about this?

**Comment by ojeytonwilliams** [comment]
I think we'd need to email them to tell them that they've been banned from -chapter-name- and are no longer invited to -list-of-events-.

**Comment by allella** [comment]
Yeah, if it was just a ban of a junk account, then maybe not. But, since it could be a real person with an RSVP, it seems appropriate.

Perhaps in a post-MVP there could be an option for the person initiating the ban to not send a message, like if they know it's a fake / bot / junk account.
